MdePkg/Library/UefiDebugLibDebugPortProtocol/DebugLibConstructor.c

```diff
@@ -34,6 +34,7 @@ EFI_BOOT_SERVICES  *mDebugBS;
   @param  Context      Pointer to the notification function's context.
 
 **/
+STATIC

```

Comment:
recommend using the C keyword 'static' and reduce use of STATIC macro as these changes are made.

Also recommend changing the name of the notification function to prefix it with the name of the lib instance.  (e.g. UefiDebugLibDebugPortProtocolExitBooServicesCallback()).  That will also help with name collisions and will also help when looking at call stacks to know which lib/module the notification is in.

Reply:
I made both of those updates.

Reply:
In order for this to work, I had to make this change https://github.com/tianocore/edk2/pull/5896.

Reply:
That's great!